src/vega_viewer/vega/specs.cljs

```diff
@@ -10,23 +10,27 @@
    :scales [{:name "category"
              :type "ordinal"
              :domain {:data "entries" :field "category"}
-             :bandWidth bar-height
+             :bandWidth (+ 3 bar-height)
              :range "height"}
             {:name "frequency"
              :type "linear"
              :range "width"
              :round true
              :domain {:data "entries" :field "frequency"}}]
-   :axes [{:scale "frequency" :type "x"}
+   :axes [{:scale "frequency"
+           :type "x"
+           :properties {:labels {:text {:template "{{datum.data}}"}}}}
           {:scale "category" :type "y"}]
    :marks [{:from {:data "entries"}
             :type "rect"
-            :properties {:enter {:y {:scale "category" :field "category"}
-                                 :height {:value 30
-                                          :offset -1}
+            :properties {:enter {:y {:scale "category"
+                                     :field "category"
+                                     :offset 3}
+                                 :height {:value bar-height
+                                          :offset -1.5}
```

Comment:
Yeah, this should be a constant. Was just messing with values. This should have been a DNM. I'll fix it.